content/getting-started/registering.md

```diff
@@ -4,8 +4,67 @@ prev: "/getting-started/"
 -->
+
+# Registering
+
+So you decided to join? That's great! There's only a couple of steps to create new account and start solving tasks, learn, and improve your skills!
+
+## First step
+
+First, you need to go to [Sign Up](https://www.codewars.com/join) page, if you haven't yet. You will be presented with a board of many languages to choose from:
+
+![languages board](./img/registering_01_choose_language.png)
+
+Choose whatever language you like, you want to learn, or are familiar with. I want to improve my skills with functional programming, and I already know a bit if Java, so I choose Scala as my initial language. Here you can pick any language available, your selection does not determine anything yet, and you will be always able to choose another language later, if you wish.
+
+## Your first challenge
+
+After selecting the language, you are presented with your very first challenge. It's an entry level task which you need to solve to prove your dedication and commitment to learning, and prove yourself worthy of training among CodeWarriors. It should not be difficult, let's see...
+
+![first task](./img/registering_02_multiply_initial.png)
+
+Task says: "The code does not execute properly. Try to figure out why." Let's take a closer look at it... Of course! Some tiny detail is missing, and it causes compilation error! Let's put it in place, fix the code, and click `SUBMIT`. Server will process our solution and... after a short while, tests report is presented to you. If you made a mistake and your answer is not correct, don't worry! Just click `TRY AGAIN` and fix your solution.
+
+![retry](./img/registering_03_multiply_wrong_answer.png)
+
+If your answer is correct, you will experience the best little thing in a life of a coder _[citation needed]_: **green tests**! Congratulations!
+
+![success](./img/registering_04_multiply_success.png)
+
+## Your new account
+
+You have proved yourself worthy. You have proved yourself committed and willing to learn and improve your skills. You are allowed to enter Codewars dojo. To get your account created, fill in a couple of details, or use your GitHub account to authenticate on Codewars:
+
+![enlist](./img/registering_05_enlist.png)
+
+Great, your account is created!
+
+**Important note:** Codewars will send you an email to confirm your email account. Go to your inbox and click on confirmation link in the message titled `"Confirm your account... Dojo Access Link"`, received from Codewars. Until you do this, you will be allowed to train and solve tasks, but some actions will be restricted or unavaliable for you. For example, you won't be able to create discourse posts. If you cannot find the email, check `Spam` folder of your inbox! If you do not confirm your account right now, you can always ask Codewars to resend you the confirmation email by clicking dedicated button in your account panel (we will get to your account panel soon).
+
+## Your training routine
```

Comment:
This section should go to "Setting Up".

Reply:
I was thinking the same while writing,gave it a bit of thought, and decided to keep it here deliberately. Thing is that initial training setup is a part of sign-up flow and a user cannot skip it or navigate out of it. It feels as integral part of sign up, as opposed to setting up your account after it's created. Having it here lets a new user to go through all the sign up process while reading a single page. Moving it to some other page will make user navigate to other place even before process feels finished and they are able to use their account.
On the other hand, training setup would have to be somehow covered, or referenced, in Setting Up part, and parts of the description can turn out to be redundant. But until I have that page ready and will be able how/if it can be referenced or mentioned, I would like to keep it here. Is this OK?

Reply:
Hmm, I think mentioning that the initial training setup exists and having a link to more details (next page) is natural, but we can improve it later.

> But until I have that page ready and will be able to see how/if it can be referenced or mentioned, I would like to keep it here. Is this OK?

Sure